### PR TITLE
Add `air_private_input` flag to cairo1-run

### PR DESCRIPTION
**Pull request by fmoletta**
Depends on (and includes changes from) #1552

### CODE REVIEW

**Review by juanbono — DISMISSED** [review]

File: cairo1-run/README.md [hunk 1]
```diff
@@ -16,16 +17,50 @@ make test
...
+cargo run ../cairo_programs/cairo-1-programs/with_input/array_input_sum.cairo --layout all_cairo --args '2 [1 2 3 4] 0 [9 8]'
+
+```
+
 To execute all the cairo 1 programs inside `../cairo_programs/cairo-1-programs/` and generate the corresponding trace and the memory files
+
 ```bash
 make run 
-``` 
+```
```
Comment (pefontana) [inline cairo1-run/README.md:48]: Nice doc!

File: Makefile [hunk 2]
```diff
@@ -287,8 +289,8 @@ compare_memory: $(CAIRO_RS_MEM) $(CAIRO_MEM)
 compare_trace_memory_proof: $(COMPILED_PROOF_TESTS) $(CAIRO_RS_TRACE_PROOF) $(CAIRO_TRACE_PROOF) $(CAIRO_RS_MEM_PROOF) $(CAIRO_MEM_PROOF)
 	cd vm/src/tests; ./compare_vm_state.sh trace memory proof_mode
 
-compare_all_proof: $(COMPILED_PROOF_TESTS) $(CAIRO_RS_TRACE_PROOF) $(CAIRO_TRACE_PROOF) $(CAIRO_RS_MEM_PROOF) $(CAIRO_MEM_PROOF) $(CAIRO_RS_AIR_PUBLIC_INPUT) $(CAIRO_AIR_PUBLIC_INPUT)
-	cd vm/src/tests; ./compare_vm_state.sh trace memory proof_mode air_public_input
+compare_all_proof: $(COMPILED_PROOF_TESTS) $(CAIRO_RS_TRACE_PROOF) $(CAIRO_TRACE_PROOF) $(CAIRO_RS_MEM_PROOF) $(CAIRO_MEM_PROOF) $(CAIRO_RS_AIR_PUBLIC_INPUT) $(CAIRO_AIR_PUBLIC_INPUT) $(CAIRO_RS_AIR_PRIVATE_INPUT) $(CAIRO_AIR_PRIVATE_INPUT)
```
Comment (pefontana) [inline Makefile:292]: The CI runs this no?

**Review by pefontana — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (fmoletta) [reply Makefile:292]: It runs the standalone script, added the necessary flags & arguments, nice catch!

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,21 +2,35 @@
 
 #### Upcoming Changes
 
+<<<<<<< HEAD
```
Comment (pefontana) [inline CHANGELOG.md:5]: jejeje i think you forgot the conflicts in the Changelog

**Review by pefontana — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (fmoletta) [reply CHANGELOG.md]: Oops

Comment (fmoletta) [reply CHANGELOG.md]: Fixed

**Review by pefontana — APPROVED** [review]
Nice!

**Review by juanbono — APPROVED** [review]

**Pull request merged**